packages/demoing-storybook/src/build/build.js

```diff
@@ -151,12 +152,26 @@ async function buildManager({ outputDir, assets }) {
   await rollupBuild(configs[1]);
 }
 
-async function buildPreview({ outputDir, assets, storyFiles, rollupConfigDecorator }) {
+async function buildPreview({
+  outputDir,
+  assets,
```

Comment:
nit: looks like we only take the `iframeHTML`. Can we destructure here in the params?

Reply:
I think deeply nested destructuring is not very readable :)

Reply:
In this function the parameters are already multiline, so I guess it's fine here. added it :)